### ISSUE TITLE
Not reloading browser (WSL)

### ISSUE THREAD

**Issue by smth** [BrowserSync/browser-sync#1817]
### Issue details

Browsersync is failing to reload my browser. I'm attempting to use it for the first time on a new machine, running WSL. In the simplest case this manifests as the browser not reloading when files are changed. I have also tried running a couple of different static site generators (11ty and Jigsaw), in both cases the servers become unresponsive after a file is changed, hanging at [reloading browser]. (I have used both of these tools successfully on another machines running WSL).

### Steps to reproduce/test case

1. Create `index.html`
2. Create `css/styles.css`
3. Run `browser-sync start --server --files "css/*.css"`
4. Edit `css/styles.css`

### Please specify which version of Browsersync, node and npm you're running

- Browsersync [  2.26.13  ]
- Node        [  8.17.0  ]
- Npm         [  6.13.4  ]

### Affected platforms

- [x] linux
- [x] windows
- [ ] OS X
- [ ] freebsd
- [ ] solaris
- [ ] other _(please specify which)_

### Browsersync use-case

- [x] API
- [ ] Gulp
- [ ] Grunt
- [x] CLI

### If CLI, please paste the entire command below
```bash
browser-sync start --server --files "css/*.css"
```

### for all other use-cases, (gulp, grunt etc), please show us exactly how you're using Browsersync

- https://github.com/11ty/eleventy/
- https://github.com/tighten/jigsaw

**Comment by C0casio45** [comment]
I have the same issue on windows. The server launch well, but nothing append when I save my project( the server see the changer but it does not refresh my browser (chome, brave)

**Comment by smth** [comment]
Are you using WSL2 by any chance @C0casio45? I've found that was the cause of this and other similar problems. Switching to WSL1 might be a solution.

**Comment by lsze** [comment]
I was able to fix a ton of unexpected Browser-sync behavior and slowness in WSL2 by making sure all my projects are under my WSL home user path `/home/user/` rather than under a Windows path `/mnt/c/`.

**Comment by smth** [comment]
Yeah, it seems that's when WSL2 becomes problematic, if you want to work from the Windows file system. So keeping everything within Linux is another (possibly better) solution.

**Comment by cecilemuller** [comment]
tbh this is why I remain with the Hyper-V backend for Docker Dektop (despite I tried to move to WSL2 and devcontainers several times), it just works and doesn't break file change events, so I can keep the sources in the normal filesystem (and on any disk, not just on the primary OS disk) while dependencies are installed & run in the container thanks to Compose, so no linux version of node_modules gets written to the Windows filesystem).

I don't know how WSL2 can expect people to move all their files to the small virtual filesystem.

**Comment by yuripallada** [comment]
@smth Not sure which IDE you're using, but if you're using Visual Studio Code and want to have the files in WSL2, you can install the `Remote - WSL` extension in visual studio code.
1. Open VScode on your windows machine
2. Install the Remote - WSL extension
3. Go to the WSL2 filesystem and the folder you want to open with code
4. type `code .`
5. WSL2 will install VS Code server, and open VS Code on your local machine. The directory that it opens is directly linked to the directory in WSL2, which means you can go ahead and do `yarn start` in your WSL2 terminal, open the browser with browsersync, and edit code in your VS Code window on the host machine, and it will perfectly reload the browser when you save code in VS Code now!

I hope this helps, it works for me!

**Comment by ARHoss** [comment]
> @smth Not sure which IDE you're using, but if you're using Visual Studio Code and want to have the files in WSL2, you can install the `Remote - WSL` extension in visual studio code.
> 
> 1. Open VScode on your windows machine
> 2. Install the Remote - WSL extension
> 3. Go to the WSL2 filesystem and the folder you want to open with code
> 4. type `code .`
> 5. WSL2 will install VS Code server, and open VS Code on your local machine. The directory that it opens is directly linked to the directory in WSL2, which means you can go ahead and do `yarn start` in your WSL2 terminal, open the browser with browsersync, and edit code in your VS Code window on the host machine, and it will perfectly reload the browser when you save code in VS Code now!
> 
> I hope this helps, it works for me!

Thanks -this worked for me
